src/main/java/games/strategy/engine/data/properties/NumberProperty.java

```diff
@@ -63,9 +63,7 @@ public JComponent getEditorComponent() {
   public boolean validate(final Object value) {
     if (value instanceof Integer) {
       final int i = ((Integer) value);
```

Comment:
Not directly related, but the outer parentheses can be removed, and the value can be cast to int directly